### PR TITLE
[14.0][MIG] account_invoice_import

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

**Comment by alexis-via** [comment]
#242

**Review by rconjour — CHANGES_REQUESTED** [review]
LGTM - but maybe link in the description to the example of the mass import bash script.

File: account_invoice_import/wizard/account_invoice_import.py [hunk 1]
```diff
@@ -0,0 +1,1448 @@
...
+        #  subtype_id = thread._creation_subtype().id
+        # AttributeError:
+        # 'account.invoice.import' object has no attribute '_creation_subtype'
+        # This traceback doesn't block the creation of the invoice.
+        # I could move this code to account.move, but I don't want to
+        # create an invoice on every email received, I still want to return
+        # nothing when there is no attachment or when the attachment
+        # fails to import or when the invoice is already created
+        # So, for the moment, we leave it like that...
+        # If you have a better idea, please suggest!
```
Comment (StefanRijnhart) [inline account_invoice_import/wizard/account_invoice_import.py]: I've implemented the obvious solution here: https://github.com/akretion/edi/pull/4, letting this model inherit from mail.thread, and it works. Maybe you have considered it before, and there may be good reasons not to use it that I've missed?

**Review by PieterPaulussen — APPROVED** [review]
LGTM

**Review by fshah-initos — APPROVED** [review]
LGTM

File: account_invoice_import/wizard/account_invoice_import.py (hunk 1 above)
Comment (alexis-via) [reply account_invoice_import/wizard/account_invoice_import.py]: Thanks !

Comment (StefanRijnhart) [reply account_invoice_import/wizard/account_invoice_import.py]: Thanks for taking it! I see now that I did not remove this comment in the proposed PR. Can you squash that in somewhere?

**Comment by alexis-via** [comment]
@rconjour I added a note about it via USAGE.rst

File: account_invoice_import/wizard/account_invoice_import.py (hunk 1 above)
Comment (StefanRijnhart) [reply account_invoice_import/wizard/account_invoice_import.py]: And a rebase while you are at it, to squash out the merge commit from my intermediate PR

**Review by rconjour — APPROVED** [review]

**Comment by mylbco** [comment]
I see the PR is approved, can it be merged @StefanRijnhart ?

**Comment by StefanRijnhart** [comment]
@mylbco a merge commit still needs to be squashed out.

**Review by StefanRijnhart — APPROVED** [review]

**Pull request merged**